signer/web/package.json

```diff
@@ -33,6 +33,29 @@
     "url": "https://github.com/ethcore/parity-sysui-app/issues"
   },
   "homepage": "https://github.com/ethcore/parity-sysui-app#readme",
+  "peerDependencies": {
```

Comment:
But it's also standalone app now. This setup breaks compilation of Signer UI app for parity.